src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -363,10 +385,13 @@ private function createFirewall(ContainerBuilder $container, $id, $firewall, &$a
         // Authentication listeners
         list($authListeners, $defaultEntryPoint) = $this->createAuthenticationListeners($container, $id, $firewall, $authenticationProviders, $defaultProvider, $configuredEntryPoint);
 
+        $config->replaceArgument(6, $configuredEntryPoint ?: $defaultEntryPoint);
```

Comment:
Good idea.
Could you give me your opinion about the `FirewallConfig::$listenerConfigs` property?
I'm not sure it is useful, values cannot be the final ones so it is pretty much a dump of what is written in yaml. Plus, I really don't see how I should display them into the profiler.